ml-agents/mlagents/trainers/tests/test_rl_trainer.py

```diff
@@ -47,7 +47,8 @@ def create_mock_policy():
 
 @mock.patch("mlagents.trainers.rl_trainer.RLTrainer.add_policy_outputs")
 @mock.patch("mlagents.trainers.rl_trainer.RLTrainer.add_rewards_outputs")
-def test_rl_trainer(add_policy_outputs, add_rewards_outputs):
+@pytest.mark.parametrize("num_vis_obs", [0, 1], ids=["", "viz"])
```

Comment:
is it possible to have the num_vis_obs also be 2 ? Or is that too much work ?

Reply:
Added, it's a one-number change